src/pfs/pfs.proto

```diff
@@ -300,6 +300,20 @@ message CreateBranchRequest {
   bool new_commit_set = 5; // overrides the default behavior of using the same CommitSet as 'head'
 }
 
+message SearchForFileInBranchRequest {
+  Commit start = 1;
+  string file_path = 2;
+  int32 limit = 3; // Does it make more sense to define a limit in terms of raw commits to search or in terms of the length of the returned 'found commits' array?
+  google.protobuf.Duration timeout = 4;
+}
+
+message SearchForFileInBranchResponse {
+  repeated Commit found_commits = 1;
+  Commit last_searched_commit = 2;
+  google.protobuf.Duration duration = 3;
```

Comment:
The user can time duration themselves and it will will be more accurate from their POV than anything we time on our end.